src/components/shared/Search/search.module.scss

```diff
@@ -1,7 +1,7 @@
 @import "../../../assets/variables";
 
 .input {
-  width: 195px;
+  width: 250px;
```

Comment:
Don't you think it will affect this component in another place?

Reply:
Don't know
Just removed it because transition effect breaks click on the search button
Didn't have time to discover the problem and solve it

Reply:
How does transition breaks click on the search button?

Reply:
You click the button
Input change its width but onClick event doesn't' work

Reply:
Are you sure? As I have just checked it, and it works correctly.